src/app/App.tsx

```diff
@@ -69,6 +70,9 @@ function DropZoneArea() {
     { id: '1h,1h', title: '1H,1H', content: 'Hello, World! [c]' },
     { id: '1h,13c', title: '1H,13C', content: 'Hello, World! [d]' },
   ];
+  function saveHandler() {
+    dispatch({ type: 'SAVE_AS_IUM' });
+  }
```

Comment:
Please implement the download logic here, not in the reducer. Please implement the download without the `file-saver` library.

It's relatively easy to do, something like this:

```ts
export function download(data: string) {
  const link = document.createElement('a');

  const blob = new Blob([data]);
  const url = URL.createObjectURL(blob);


  link.setAttribute('href', url);
  link.setAttribute('download', name);
  link.style.visibility = 'hidden';
  document.body.appendChild(link);
  link.click();
  document.body.removeChild(link);
  URL.revokeObjectURL(url);
}
```

Reply:
fixed thank you